src/admin/settings/settings.html

```diff
@@ -1,49 +1,49 @@
+<form class="settings-form" id="settingsFileForm" enctype="multipart/form-data" ng-submit="uploadSettings()">
     <fieldset>
-        <h4 class="section-title">{{'admin.settings.upload' | translate}}</h4>
+        <h4>{{'admin.settings.upload' | translate}}</h4>
         <div class="form-group">
             <label for="fileInput">{{'admin.settings.fileChosen' | translate}}</label>
             <input class="button-file" type="file" id="fileInput" name="file" value="{{'server.button.selectFile' | translate}}" admin-import="settingsFile"/>
         </div>
     </fieldset>
     <input type="submit" value="{{'admin.submit' | translate}}" />
 </form>
-<form id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
+<form class="settings-form" id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
     <fieldset>
-        <h4 class="section-title">{{'admin.settings.location' | translate}}</h4>
+        <h4>{{'admin.settings.location' | translate}}</h4>
         <div class="form-group">
             <label for="settingsLocation">{{'admin.settings.location.new' | translate}}</label>
             <input type="text" id="settingsLocation" name="settingsLocation" ng-model="location"/>
         </div>
     </fieldset>
     <input type="submit" value="{{'admin.submit' | translate}}"/>
 </form>
-<form>
+<form class="settings-form">
     <fieldset>
-        <h4 class="section-title">{{'admin.settings.importexport.header' | translate}}</h4>
+        <h4>{{'admin.settings.importexport.header' | translate}}</h4>
         <div class="form-group">
             <label for="exportConfig">{{'admin.settings.importexport.export' | translate}}</label>
             <input class="button button-export" type="button" id="exportConfig" name="exportConfig" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" />
         </div>
     </fieldset>
 </form>
 
-<h4 class="section-title">{{'admin.settings.platform' | translate}}</h4>
-<div ng-repeat="pSettings in platformSettings.settingsList">
-    <form id="platformSettingsForm">
-        <h4 class="partial-title">{{'admin.settings.section.'+ pSettings.section | translate}}</h4>
+<div>
+    <h3>{{'admin.settings.platform' | translate}}</h3>
+    <form class="settings-form" id="platformSettingsForm" ng-repeat="pSettings in platformSettings.settingsList">
+        <h4>{{'admin.settings.section.'+ pSettings.section | translate}}</h4>
         <div class="form-group" ng-repeat="option in pSettings.settings">
             <label ng-if="!isBool(option)" for="{{option.key}}">
                 {{'admin.settings.' + option.key | translate}}
-                <span class="icon-documentation" data-trigger="hover" data-placement="left" bs-popover="{{'admin.settings.tooltip.' + option.key | translate}}" />
+                <span class="icon-documentation" uib-tooltip="{{'admin.settings.tooltip.' + option.key | translate}}"/>
```

Comment:
@nickdotreid @mkruszynski The blinking tooltip can be fixed by adding `tooltip-placement="left"`. The tooltip would show on the left (same as in the Motech Core) and without any blinking.